packages/web-config-server/src/apiV1/view.js

```diff
@@ -1,10 +1,13 @@
-import { convertDateRangeToPeriods, replaceValues, CustomError } from '@tupaia/utils';
+import { convertDateRangeToPeriods, CustomError, replaceValues, utcMoment } from '@tupaia/utils';
 import { getDhisApiInstance } from '/dhis';
 import { isSingleValue } from './utils';
 import { DataAggregatingRouteHandler } from './DataAggregatingRouteHandler';
 import { DashboardPermissionsChecker } from './permissions';
 import { getDataBuilder } from '/apiV1/dataBuilders/getDataBuilder';
 
+// Same as in @tupaia/web-frontend
+const DEFAULT_MIN_DATE = '20150101';
```

Comment:
This is also defined in `web-fronted/src/components/DateRangePicker/constants.js`.

Ideally we would define this constant in a single place (eg the `utils` package), but we've had a bit of trouble loading functionality from other tupaia packages in `web-frontend`. Hopefully the above comment helps!